### PR TITLE
Api setup

### PR DESCRIPTION
**Pull request by jstreifel-33**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_LOCATION_IQ_KEY=pk.5b978d105e102fbfdc70c18da1625112
```
Comment (RivaD2) [inline .env]: HI there!

Feedback here is to get .env properly .gitignored. Add `.env` to `.gitignore`. You may find that you have to first remove env then commit, then add it to gitignore then commit. Then add env variables after etc.

File: src/components/Main.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      <>
+        <Container className="main">
+          <SearchForm queryLocation={this.props.queryLocation} getMap={this.props.getMap} latLon={this.props.latLon} />
+        </Container>
+        <Container>
+          {this.props.showResults &&
+            <DispResults location={this.props.location} mapUrl={this.props.mapUrl} show={this.props.showResults} latLon={this.props.latLon} />
+          }
+        </Container>
+      </>
```
Comment (RivaD2) [inline src/components/Main.js:18]: Nice job on separating out logic into different components!

Feedback here is really stylistic choice I suppose: If you have more than 3 or so props, feel free to indent for easier readability:

```
<DispResults
   location={this.props.location} 
   mapUrl={this.props.mapUrl} 
   show={this.props.showResults} 
   latLon={this.props.latLon} 
  />
```

Some eslint rules in codebases have line length limits etc. 👍

File: .env (hunk 1 above)
Comment (RivaD2) [reply .env]: I see you haven't merged PR yet, so go ahead and fix env before merge :)

Comment (jstreifel-33) [reply .env]: oops, good catch!

Comment (RivaD2) [reply .env]: I saw you fixed it so nice work :) 👍

**Pull request merged**